src/lib.rs

```diff
@@ -448,21 +448,21 @@ fn to_rgb_value(value: f32, temp_1: f32, temp_2: f32) -> f32 {
             converted = temp_1;
         }
     } else {
-        converted = (temp_2 + (temp_1 - temp_2)) * value * 6.0;
+        converted = temp_2 + ((temp_1 - temp_2) * value * 6.0);
     }
 
     converted
 }
```

Comment:
Oh this logically makes a lot more sense/is easier to read 👍

Reply:
Godfrey: looks like this is still outstanding, I was suggesting that we can just flatten it into the second version. If you look at the annotations, I believe they are equivalent. (Otherwise hopefully some tests will fail.)

Reply:
OH I missed the fact that you were suggesting the second version (I understood that it was equivalent but didn't realize that's one one you were suggesting). Fixing now.